slide-deck.js

```diff
@@ -280,11 +316,21 @@ class slideDeck extends HTMLElement {
     this.slides = this.querySelectorAll(':scope > :not([slot])');
     this.slideCount = this.slides.length;
     this.style.setProperty('--slide-count', this.slideCount);
+    this.style.setProperty('--slide-count-string', `'${this.slideCount}'`);
 
     this.slides.forEach((slide, index) => {
       const slideIndex = index + 1;
       slide.id = this.#slideId(slideIndex);
       slide.style.setProperty('--slide-index', slideIndex);
+      slide.style.setProperty('--slide-index-string', `'${slideIndex}'`);
+
+      slide.querySelectorAll(':scope button[slide-activate]').forEach((btn) => {
+        btn.addEventListener('click', () => {
+          this.goTo(slideIndex)
+          const inView = btn.getAttribute('slide-activate');
+          if (inView) { this.setAttribute('slide-view', inView); }
```

Comment:
There's not currently, because this allows you to define and set any layouts you want. The alternative approach, if we want validation, would be allowing you to register additional views - and then we would only let you set a view that's been registered?